mgmt_cli_test.py

```diff
@@ -1147,3 +1147,24 @@ def test_suspend_and_resume_without_starting_tasks(self):
                 f'Task {current_task_status} did not remain in "{TaskStatus.STOPPED}" status, but instead ' \
                 f'reached "{current_task_status}" status'
         self.log.info('finishing test_suspend_and_resume_without_starting_tasks')
+
+    def test_manager_installed_and_functional(self):
+        """Verify that the Manager is installed and functional.
+
+        The test is intended for execution on non-main OS distributions (for example, debian10)
+        where the main goal is to execute installation test.
+        The rest of the checks are executed on Ubuntu distribution in test_manager_sanity.
+        """
+        self.log.info('starting test_manager_installed_and_functional')
+
+        manager_node = self.monitors.nodes[0]
+        scylla_node = self.db_cluster.nodes[0]
+
+        # Check scylla-manager and scylla-manager-agent are up and running (/ping endpoint)
+        manager_node.is_manager_server_up()
+        scylla_node.is_manager_agent_up()
```

Comment:
shouldn't the test also verify communication between agent and server? 
Or at least cluster is registered in server?
I believe several short functional checks could find some regressions without sacrificing a lot of time.

Reply:
yes, makes sense, added several checks on cluster addition and health check.

Reply:
please retry test builds if everything passes

Reply:
Yes, it works, verified here:
https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/mikita/job/sct/job/manager-master/job/installation-tests/job/ubuntu22-installation/4/